mindmeld/components/_elasticsearch_helpers.py

```diff
@@ -223,13 +223,29 @@ def delete_index(
         raise KnowledgeBaseError
 
 
+def create_index_mapping(base_mapping, mapping_data):
```

Comment:
this function is not being used anywhere

Reply:
It's being used in question_answerer.py